ext/native/net/http_client.cpp

```diff
@@ -126,7 +126,7 @@ void DeChunk(Buffer *inbuffer, Buffer *outbuffer, int contentLength, float *prog
 		if (!line.size())
 			return;
 		int chunkSize;
-		sscanf(line.c_str(), "%x", &chunkSize);
+		sscanf(line.c_str(), "%x", (unsigned int *)&chunkSize);
```

Comment:
I think a cast of a pointer argument to `sscanf()` should almost be an error.  In this case, maybe `dechunkedBytes` and `chunkSize` would just both be `uint32_t`s?

-[Unknown]